kratos/modified_shape_functions/triangle_2d_3_ausas_incised_shape_functions.cpp

```diff
@@ -0,0 +1,152 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
```

Comment:
```suggestion
//  License:             BSD License
//                            Kratos default license: kratos/license.txt
//
//  Main authors:    Franziska Wahl
```
Header format and authorship.